mkdocs.yml

```diff
@@ -94,5 +94,6 @@ nav:
       - Migration to Airlift: migration-to-airlift.md
   - Contribute:
     - Code: development.md
+    - Docker: docker.md
```

Comment:
Maybe it should go in the installation section already now.. or in the follow up work as trimmed down separate page.